docs/topics/cache.txt

```diff
@@ -820,7 +827,9 @@ return if the object doesn't exist in the cache::
 
 To add a key only if it doesn't already exist, use the ``add()`` method.
 It takes the same parameters as ``set()``, but it will not attempt to
-update the cache if the key specified is already present::
+update the cache if the key specified is already present:
+
+.. method:: cache.add(key, value, timeout)
```

Comment:
These lines should go above the paragraph that describe its behavior.

Reply:
Done, should I follow the same for rest of the methods as well ?

Reply:
Yes please.

Reply:
done